### ISSUE TITLE
Status Carrying through death despite config?

### ISSUE THREAD

**Issue by Omuck3** [Funwayguy/EnviroMine#150]
Whenever I die now my sanity/thirst don't reset... :(

Config says that won't happen.

**Comment by GenDeathrow** [comment]
Im not sure whats happening, I looked into this, but the code looks good.

https://github.com/EnviroMine/EnviroMine-1.7/blob/master/src/main/java/enviromine/handlers/EM_EventManager.java#L244-L252

**Comment by GenDeathrow** [comment]
are you using cauldron

**Comment by PitchBright** [comment]
I just changed my server and client config to
    B:"Keep statuses on death"=true

And that seemed to fix this bug, although I haven't tested it extensively.

Could it be that you have this worded incorrectly in the description in the config, GenDeathrow?

**Comment by GenDeathrow** [comment]
I am not sure I think funwayguy added this in .. maybe thislooksfun. it was a while ago.. either way.. if the 
if(player died && (keep status=false)) than {reset status to default status}
else if (player died) than {clap status to the edge of safe range }

I'll do some play testing on it check it out. I really want to try and stabilize some things on EnviroMine.

**Comment by PitchBright** [comment]
If it turns out it was just a simple case of true being described as false, and false being described as true… I think a lot of people's problems will be solved  :+1:

**Comment by rob-bb** [comment]
Did anything happen to this?  Was it reversed?

**Comment by PitchBright** [comment]
what do you mean @rob-bb ?

**Comment by rob-bb** [comment]
The comments suggest the setting B:"Keep statuses on death"=true might have been incorrect with true actually being false?   

Was just curious as I spent a couple of hours today working out a workaround to this as I believed it was a Cauldron issue as per this issue on the other enviromine github.  https://github.com/EnviroMine/EnviroMine-1.7/issues/181

**Comment by PitchBright** [comment]
Oh I see.

Did you try my suggestion of setting it to True?

**Comment by rob-bb** [comment]
No, I just discovered this after my complex workaround.  Might look at it tomorrow but I guess since it has been a few months it might have been fixed if it was broken.

**Comment by PitchBright** [comment]
They haven't worked on the mod in ages, AFAIK. So I wouldn't bank on that.

Please try it tomorrow and let us know here, how you made out with it. It'd really help us troubleshoot the problem (which I don't seem to have and I wonder if it's because of my settings).

**Comment by rob-bb** [comment]
Ok, I just tried it after disabling my workaround, and setting to true is still true and caries over the effects after player death on KCauldron 168.

**Comment by PitchBright** [comment]
Yer right then, must be a Cauldron thing.

@Omuck3 are you using Cauldron?

**Comment by Omuck3** [comment]
I was, yes.

**Comment by hanakocz** [comment]
I had similar problem with my mod , the thing is that reset of stat don't work on Cauldron if it is binded to deathEvent, because Cauldron does some strange things here. 

However setting the reset of stats to PlayerEvent.PlayerRespawnEvent fixed it even for Cauldron
